### ISSUE TITLE
[TODO] Create server for Node & Express

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/radio-logger#32]
Create a Node/Express server for the REACT version of this application.

I inserted a server.js which would need to be completed for this to work.

Also, implement Axios HTTP client and nodemon monitoring server for development mode.

**Comment by gbowne1** [comment]
[TODO] Complete routing in Node/Express server.js

**Comment by 1pervysage0** [comment]
You want the routing to be done?
Can you give me a brief of what you want?
i'll love to do this task.

**Comment by gbowne1** [comment]
I will try @1pervysage0..

I can imagine that most of the people using this app will view it on desktop and the occasional tablet.

This is the way I see things going..
 
On loading the initial page (not logged in), Ideally I'd like to see a splash page (perhaps a modal with the app title and perhaps a carousel of pictures) then directed to a login and or a signup/register (if the user is a new user) if not then just to login..

If a returning user has forgotten his/her username and/or password there should be a create new password thing or a function to send the user his/her username and/or password.

Once logged in the new user sees a create profile page, allowing the user to enter in details.  A returning user should just enter their login details (if its not already autocompleted) and login. Add a function so that the user can see his/her password (usually under a :eye: next to the field.  Possibly also having a "Remember me" checkbox.  

Then we have to check to see if the username and password were correct with some Authentication.

I've read there are several ways to do this.  with Auth0, OAuth, OAuth2, Axios... I never decided exactly which method to use.

I am thinking that for most people using this app, authentication lasts only for one session,

The page should render as it is now, the main App page being routed to "/"

A user may want to change some settings.  This should be accessed through the Settings component with /Settings.js
 when clicking on the settings.  I think mainly this will be used especially for Notification settings.

A user may also want to enter some data into the logging fieldsets, clicking on Logging on the left Nav drawer.  this would open a /log or /logging route, with whatever child route based on nav menu choice.  Then the user would select which type of logging from the menu, and then to the various components.  SwlLog.js if the user chooses 'Shortwave Logging' and it would render to /swllog.  Clicking on Amateur Radio Logging would or going to /hamlog would render HamLog.js conmponent.

Once finished the user ideally should click a button to logout.

**Comment by 1pervysage0** [comment]
Ok I'll workout on it.
And will keep you updated if any blocker comes.

**Comment by gbowne1** [comment]
@1pervysage0 yeah please let me know if you need any other imformation that will help.
